src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/emails/RpaEmailTemplates.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.claimstore.config.properties.emails;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+@Component
+public class RpaEmailTemplates {
+    public String getClaimIssuedEmailSubject() {
+        return readString("/rpa/templates/email/claimIssued/subject.txt");
+    }
+
+    public String getClaimIssuedEmailBody() {
+        return readString("/rpa/templates/email/claimIssued/body.txt");
+    }
+
+    private String readString(String resourcePath) {
```

Comment:
`new String(Files.readAllBytes(Paths.get(url)), UTF_8);` can do the same job we are doing with `readBytes(...)` and `readString(...)`

Reply:
I will leave that one to you @sabahirfan. The other problem is that `readString` is duplicated in `uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailTemplates` as well.

Reply:
sure, will refactor that in separate PR.